src/elements-experimental/seat-reservation/seat-reservation-area/seat-reservation-area.component.ts

```diff
@@ -0,0 +1,66 @@
+import { forceType } from '@sbb-esta/lyne-elements/core/decorators.js';
+import type { CSSResultGroup, PropertyValues, TemplateResult } from 'lit';
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { type AreaBackgroundState, type ElementMounting } from '../seat-reservation.js';
+
+import style from './seat-reservation-area.scss?lit&inline';
+
+/**
+ * Visualize an area with a special meaning within a wagon.
+ */
+export
+@customElement('sbb-seat-reservation-area')
+class SbbSeatReservationAreaElement extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  /** Rotation Prop */
```

Comment:
already on non very important level but property descriptions could be improved.